### PR TITLE
Vte fixes ss

### PR DESCRIPTION
**Pull request by srevinsaju**
@Saumya-Mishra9129 I am working on this

### CODE REVIEW

**Review by Saumya-Mishra9129 — COMMENTED** [review]
Can you suggest alternative for fork_command() .I looked and found spawn_async() but not tested it yet.

File: frotz.py [hunk 1]
```diff
@@ -13,26 +13,35 @@
...
-gi.require_version('Vte','2.91')
 import logging
+import configparser
+import os.path
+import platform
+import dbus
+
+gi.require_version('Gtk', '3.0')
+gi.require_version('Gdk', '3.0')
+gi.require_version('Vte', '2.91')
```
Comment (Saumya-Mishra9129) [inline frotz.py:36]: Should we include version 2.90 also here??

**Comment by srevinsaju** [comment]
Added `spawn_sync`, function instead of `spawn_async`. I assumed it would be better; 
Please do a `rebase` commit and not a `merge` to avoid a `merge` commit 

@Saumya-Mishra9129 at the moment, most of the stuff work, however 
the main stuff `frotz` does not work because the library has been compiled on a older version

Please recompile `frotz` on an older version of `manylinux2014` device so that it works on all devices

Reproducer
```
cd bin
./frotz
```

**Comment by Saumya-Mishra9129** [comment]
> @Saumya-Mishra9129 at the moment, most of the stuff work, however
the main stuff frotz does not work because the library has been compiled on a older version
> Please recompile frotz on an older version of manylinux2014 device so that it works on all devices
> Reproducer
> cd bin
./frotz

Thanks I will do.

**Pull request merged**